src/errors.js

```diff
@@ -4,9 +4,11 @@ export class MessageError extends Error {
   constructor(msg: string, code?: string) {
     super(msg);
     this.code = code;
+    this.isMessageError = true;
```

Comment:
Can we use an `instanceof` check instead?

Reply:
Yeah I was thinking about using `instanceof`, but what if subclasses of `MessageError` are thrown? Should I check for all of them or only `MessageError`?

Reply:
Only MessageError is fine. If they're subclass, the instanceof will work as expected and will return true, even if they are from a derived class.